src/components/studies/app-design/GeneralContactAndSupportSection.tsx

```diff
@@ -1,21 +1,27 @@
 import React from 'react'
 import {makeStyles} from '@material-ui/core'
 import Subsection from './Subsection'
-import {FormControl, Box, FormHelperText} from '@material-ui/core'
+import {FormControl, Box} from '@material-ui/core'
 import {Contact} from '../../../types/types'
 import clsx from 'clsx'
 import {isInvalidPhone, isValidEmail} from '../../../helpers/utility'
 import {makePhone} from '../../../helpers/utility'
 import FormGroupWrapper from './FormGroupWrapper'
 import TextInputWrapper from './TextInputWrapper'
 import {ContactType} from './AppDesign'
+import AlertWithTextWrapper from '../../widgets/AlertWithTextWrapper'
 
 const useStyles = makeStyles(theme => ({
   firstFormElement: {
     marginTop: theme.spacing(2.5),
   },
   errorText: {
-    marginTop: theme.spacing(-0.5),
+    marginTop: theme.spacing(-1.5),
```

Comment:
You are duplicating the class in another component. Can this class live in the AlertWithTextWrapper?